# Goal
string robot_name
geometry_msgs/PoseStamped pose
bool pick
geometry_msgs/Vector3 grasp_offset
geometry_msgs/Vector3 approach_offset
bool liftup_after
float32 speed_fast
float32 speed_slow
---
# Result
uint8 SUCCESS=0
uint8 MOVE_FAILURE=1
uint8 APPROACH_FAILURE=2
uint8 DEPARTURE_FAILURE=3
uint8 PICK_FAILURE=4

int32 result
---
# Feedback
geometry_msgs/PoseStamped current_pose
